doc/src/guides/solving.md

```diff
@@ -0,0 +1,45 @@
+# Solve Using the SymPy Python Library
+
+The Python package SymPy can symbolically solve equations, differential equations, 
+linear equations, nonlinear equations, matrix problems, inequalities, 
+Diophantine equations, and evaluate integrals. SymPy can also solve numerically.
+
+Learn how to use SymPy computer algebra system to:
+
+| Description                                                  | Example                                                                                                                     |
+|--------------------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------|
+| [ Solve an equation algebraically ](http://docs.sympy.org)                        | [  $x^2 = 4 \to x =[-2, 2] $  ](http://docs.sympy.org)                                                                                                       |
```

Comment:
Since we're using math notation, let's use set notation `\{-2, 2\}` so that this doesn't look like an interval.